etc/singularity.conf

```diff
@@ -17,6 +17,11 @@ allow setuid = yes
 # Should we allow users to request the PID namespace?
 allow pid ns = yes
 
+# ALLOW PRIVILEGED FOPEN: [BOOL]
+# DEFAULT: no
+# Should singularity try to fopen container images in privileged mode
+# if the calling user does not have sufficient permissions?
```

Comment:
Since this is a pretty scary comment, can you please elaborate more about what protections are still in place?